src/test/java/de/tum/cit/aet/artemis/core/security/jwt/JWTFilterTest.java

```diff
@@ -61,6 +61,40 @@ void testJWTFilter() throws Exception {
         assertThat(SecurityContextHolder.getContext().getAuthentication().getName()).isEqualTo("test-user");
     }
 
+    @Test
+    void testJWTFilterBearer() throws Exception {
+        UsernamePasswordAuthenticationToken authentication = new UsernamePasswordAuthenticationToken("test-user", "test-password",
+                Collections.singletonList(new SimpleGrantedAuthority(Role.STUDENT.getAuthority())));
+
+        String jwt = tokenProvider.createToken(authentication, false);
+        MockHttpServletRequest request = new MockHttpServletRequest();
+        request.setCookies(new Cookie(JWTFilter.JWT_COOKIE_NAME, jwt));
+        request.addHeader("Authorization", "Bearer " + jwt);
+        request.setRequestURI("/api/test");
+
+        MockHttpServletResponse response = new MockHttpServletResponse();
+        MockFilterChain filterChain = new MockFilterChain();
+        jwtFilter.doFilter(request, response, filterChain);
+        assertThat(response.getStatus()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    @Test
+    void testJWTFilterCookieAndBearer() throws Exception {
```

Comment:
I think you swapped the names of these tests. The first one has something to do with a cookie, not the second, right?